components/script/dom/webidls/MessagePort.webidl

```diff
@@ -18,6 +18,7 @@ interface MessagePort : EventTarget {
   attribute EventHandler onmessageerror;
 };
 
-dictionary PostMessageOptions {
+// Used to declare which objects should be transferred.
```

Comment:
this is only true for the `transfer` part, I think the idea is that more options could added later. Anyway I don't think we need the comment here actually, because I don't see it either at https://html.spec.whatwg.org/multipage/web-messaging.html#messageport

Reply:
Copied it from Firefox, I will remove it.

Reply:
```suggestion
```